wiki/Beatmapping/Hitsound/en.md

```diff
@@ -21,13 +21,15 @@ Beatmappers can replace any default sound samples with custom samples using a [b
 
 ## Active hitsound
 
-An active hitsound correlates to a player's clicking by reaching its peak impact immediately when played. Drum sound samples are the most frequently used active hitsounds because they provide clear and immediate feedback.
+A hitsound is considered *active* if it is on a clickable part of a hit object, for example a hit circle or slider start. These help the player judge whether they clicked early or late by playing a clear impactful sound at the time of clicking. If clicked correctly, the sound should match up with a beat in the song.
 
 ## Passive hitsound
 
-A passive hitsound does not correlate to a player's clicking. These are usually sound effects that accompany active hitsounds.
+If a hitsound is on a non-clickable part of a hit object, for example a slider repeat or spinner end, it is considered *passive*. These always play at the same time as long as the player is interacting with the hit object, unlike active hitsounds. Passive hitsounds do not need to have a clear impact or be audible, but often use the same sounds as active hitsounds when over beats in the song in order to help the player maintain rhythm.
```

Comment:
part of my concern was this wording of "when over beats in the song" that can be expressed more cleanly

```suggestion
If a hitsound is on a non-clickable part of a hit object, for example a slider repeat or spinner end, it is considered *passive*. These always play at the same time as long as the player is interacting with the hit object, unlike active hitsounds. Passive hitsounds do not need to have a clear impact or be audible, but often use the same sounds as active hitsounds when they land on distinct beats in order to help the player maintain rhythm.
```